signing/signing_test.go

```diff
@@ -0,0 +1,28 @@
+package signing
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetResults(t *testing.T) {
```

Comment:
Add tests for testing behavior when URL params contain: "../", "/", "\\/" etc.

Reply:
+1, also a test when the bucket does not exist to hit the `http.StatusInternalServerError` path